### PR TITLE
show relative preset on date button if selected

### PR DESCRIPTION
**Pull request by gau94rav**
When a preset is selected instead of custom absolute date, show that selected preset as filter label.

[Design](https://www.figma.com/file/IaywW74sLSgYJSYHjOC0ZL/Reporting-Beta-2022)

### CODE REVIEW

**Comment by sareg0** [comment]
@gau94rav can you can a screenshot of the design I should look at? I am getting taken to the wider file and I'm having trouble finding the calendar component.

File: visualizations/frontend/filters/v2/TDatePicker.vue [hunk 1]
```diff
@@ -27,18 +39,21 @@
         </button>
       </div>
 
+      <!-- To hide default clear icon -->
```
Comment (sareg0) [inline visualizations/frontend/filters/v2/TDatePicker.vue:42]: Is that comment accurate? I

Comment (gau94rav) [reply visualizations/frontend/filters/v2/TDatePicker.vue:42]: Yes I am using `<div slot="icon-clear"></div>` to hide the default icon of this component, it does not match with our design.

**Comment by gau94rav** [comment]
> Design

@sareg0 The actual calendar component is not really present in our figma design but we have this component that should show up when a preset is selected instead of an absolute date.

Screenshot 2022-09-28 at 2 11 00 PM

**Pull request merged**